src/main/java/com/blackducksoftware/integration/jira/config/controller/action/IssueCreationConfigActions.java

```diff
@@ -134,28 +150,36 @@ public BlackDuckJiraConfigSerializable updateConfig(final BlackDuckJiraConfigSer
         validateInterval(config);
         validateCreator(config);
         projectMappingConfigActions.validateMapping(config);
-        final String firstTimeSave = pluginSettingsWrapper.getFirstTimeSave();
+
+        final String username = authorizationChecker.getUsername(request).orElse(null);
+        pluginConfigurationAccessor.setJiraAdminUser(username);
+
+        final String firstTimeSave = pluginConfigurationAccessor.getFirstTimeSave();
         if (firstTimeSave == null) {
-            pluginSettingsWrapper.setFirstTimeSave(BlackDuckPluginDateFormatter.format(new Date()));
+            pluginConfigurationAccessor.setFirstTimeSave(BlackDuckPluginDateFormatter.format(new Date()));
         }
 
+        final PluginIssueCreationConfigModel previousIssueCreationConfig = globalConfigurationAccessor.getIssueCreationConfig();
+
         final String issueCreatorJiraUser = config.getCreator();
-        logger.debug("Setting issue creator jira user to: " + issueCreatorJiraUser);
-        pluginSettingsWrapper.setIssueCreatorUser(issueCreatorJiraUser);
-        pluginSettingsWrapper.setPolicyRulesJson(config.getPolicyRulesJson());
-        pluginSettingsWrapper.setProjectMappingsJson(config.getHubProjectMappingsJson());
-        final String username = authorizationChecker.getUsername(request).orElse(null);
-        pluginSettingsWrapper.setJiraAdminUser(username);
-        final Optional<Integer> previousInterval = pluginSettingsWrapper.getIntervalBetweenChecks();
+        final Optional<Integer> previousInterval = previousIssueCreationConfig.getGeneral().getInterval();
```

Comment:
previousIssueCreationConfig doesn't look like it's used anywhere else and can probably be collapsed.